charts/prl-cos/values.yaml

```diff
@@ -1,41 +1,41 @@
 java:
   applicationPort: 4044
-  ingressHost: prl-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+  ingressHost: prl-cos-{{ .Values.global.environment }}.{{ .Values.global.environment }}.platform.hmcts.net
```

Comment:
Deprecated for AAT and Preview only not for prod.

https://hmcts-reform.slack.com/archives/CA4F2MAFR/p1675868743958669

Reply:
reverted for prod